3.3/.sti/bin/assemble

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash -e
+#
+# Default STI assemble script for the python-3.3 image.
+#
+source "../lib/python_context"
```

Comment:
Please use `source scl_source enable python33`, it was recommended to us by SCL people.